libraries/chain/include/eosio/chain/wasm_module_cache.hpp

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <eosio/chain/types.hpp>  // for digest_type
+#include "IR/Module.h"
+#include <eosio/chain/multi_index_includes.hpp>
+
+#include <mutex>
+
+namespace eosio { namespace chain {
+   struct wasm_module_cache {
+      const IR::Module& get_module( const digest_type& code_hash, uint8_t vm_type, uint8_t vm_version, std::vector<U8>& bytes );
+      void code_block_num_last_used(const digest_type& code_hash, uint8_t vm_type, uint8_t vm_version, uint32_t block_num);
+      void current_lib(uint32_t lib);
+
+      struct module_entry {
+         digest_type   code_hash;
+         uint8_t       vm_type = 0;
+         uint8_t       vm_version = 0;
+         uint32_t      first_block_num_used;
```

Comment:
Does not appear to be used.

Reply:
Also does not appear to be used in wasm_interface_cache. Removing that unused index would be a performance boost.

Reply:
I don't see it used either.  As it is defined in wasm_interface_private.hpp, an implementation class, it is safe to remove. Will remove it from both files.